src/evict/evict_lru.c

```diff
@@ -260,14 +260,17 @@ __wt_evict_server_wake(WT_SESSION_IMPL *session)
     cache = conn->cache;
 
     if (WT_VERBOSE_LEVEL_ISSET(session, WT_VERB_EVICTSERVER, WT_VERBOSE_DEBUG_2)) {
-        uint64_t bytes_inuse, bytes_max;
+        uint64_t bytes_inuse, bytes_max, total_dirty_bytes, total_updates_bytes;
```

Comment:
For the sake of consistency, we could rename `total_dirty_bytes` to `bytes_dirty` and `total_updates_bytes` to `bytes_update`. 
Also, we should consider declaring this in alphabetical order. Like below -
` uint64_t bytes_dirty, bytes_inuse, bytes_max, bytes_update`